test/parallel/test-fetch.mjs

```diff
@@ -1,5 +1,3 @@
-// Flags: --experimental-fetch --no-warnings
```

Comment:
Do we have any negative tests with `--no-experimental-fetch`?

Reply:
Not yet

Reply:
Test added.